jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -208,7 +208,7 @@ public void apply(Project project) {
           jibDependencies.add(mainSourceSet.getRuntimeClasspath());
 
           jibTaskProviders.forEach(
-              provider -> provider.configure(task -> task.setDependsOn(jibDependencies)));
+              provider -> provider.configure(task -> jibDependencies.forEach(task::dependsOn)));
```

Comment:
Question: does `task.dependsOn(jibDependencies)` not work? (`dependsOn()` accepts `Object...`, but I am not actually sure if passing `List<TaskProvider<Task>>` will work as intended.)

Reply:
@chanseokoh I was curious and tried it out, but it looks like the alternative is to do:
```
task.setDependency(jibDependencies.toArray())
```
For whatever reason, the object translation doesn't work well for collections?